audio/freac/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           makefile 1.0
 
-github.setup        enzo1982 freac 1.1.3 v
+github.setup        enzo1982 freac 1.1.4 v
 github.tarball_from releases
 revision            1
```

Comment:
When increasing the `version`, the `revision` needs to be set back to 0.

Reply:
Done.